js/bloodmallet_standard.js

```diff
@@ -968,41 +973,20 @@ function update_chart() {
   while (standard_chart.series[0]) {
     standard_chart.series[0].remove(false);
   }
+  const minIlvl=Math.min(...trinketArray["Simulated itemlevels"]);
+  const baselineDps=trinketArray["trinkets"]["baseline"][minIlvl];
 
-  for (itemlevel_position in loaded_data[chosen_class][chosen_spec][data_view][fight_style]["Simulated itemlevels"]) {
+  for (let itemlevel_position in trinketArray["Simulated itemlevels"]) {
 
-    var itemlevel = loaded_data[chosen_class][chosen_spec][data_view][fight_style]["Simulated itemlevels"][itemlevel_position];
+    var itemlevel = trinketArray["Simulated itemlevels"][itemlevel_position];
     var itemlevel_dps_values = [];
 
     for (trinket of dps_ordered_trinkets) {
+      const currentItemDps=trinketArray["trinkets"][trinket][itemlevel];
+      const lastItemDps=trinketArray["trinkets"][trinket][trinketArray["Simulated itemlevels"][String(Number(itemlevel_position) + 1)]] || baselineDps;
```

Comment:
lastItemDps needs to be altered for if it is zero, then replace it with the baselineDps to fix these overly long bars for some trinkets.

const -> let
if lastItemDps == 0 -> baselineDps